cloudigrade-client/cloudigrade-api-spec.yaml

```diff
@@ -0,0 +1,53 @@
+# This is a file maintained by the rhsm-subscription project that describes
+# a portion of the Cloudigrade API.
+openapi: "3.0.2"
+info:
+  title: "cloudigrade-api"
+  description: "Third-party specification for Cloudigrade API"
+  version: 1.0.0
+
+paths:
+  /concurrent:
+    description: "Operations on concurrent usages"
+    parameters:
+    - name: limit
+      required: false
+      in: query
+      description: "Number of results to return per page."
+      schema:
+        type: integer
+    - name: offset
+      required: false
+      in: query
+      description: "The initial index from which to return the results."
+      schema:
+        type: integer
+    get:
+      summary: "List daily concurrent usages"
+      operationId: "ListDailyConcurrentUsages"
+      tags:
+        - concurrent
+      responses:
+        '200':
+          description: "The operation completed successfully"
+          content:
+            application/json:
+              schema:
```

Comment:
Doing the schema inline like this doesn't generate a model class. Can you please move to `$ref` style?